det3d/datasets/argoverse/argoverse_dataset.py

```diff
@@ -0,0 +1,106 @@
+from det3d.datasets.argoverse.argoverse_common import *
+from det3d.datasets.custom import PointCloudDataset
+from det3d.datasets.registry import DATASETS
+
+
+@DATASETS.register_module
+class ArgoverseDataset(PointCloudDataset):
+
+    def __init__(self, root_path, pipeline=None, test_mode=False):
+        self._log_id_timestamps_list = ArgoverseDataset.load_logs(root_path, 1)
+        # Not using info_path but loading ground truth directly for now
+        super(ArgoverseDataset, self).__init__(
+            root_path, None, pipeline, test_mode=test_mode
+        )
+        self._num_point_features = 3  # x, y, z
+
+    def __getitem__(self, index):
+        return self.get_sensor_data(index)
+
+    def __len__(self):
+        return len(self._log_id_timestamps_list)
+
+    def get_sensor_data(self, index):
+        log_id, timestamps = self._log_id_timestamps_list[index]
+        clouds = load_all_clouds(self._root_path, log_id, timestamps)
+        clouds = perform_SE3(clouds, self._root_path, log_id)
+        clouds = grids_group_and_SE3(clouds, self._root_path, log_id, 1)
+        points = np.float32(list(clouds.values())[0][0])
+
+        data_dict = load_all_boxes(self._root_path, log_id, timestamps)
+        bbox_dict = convert_to_boundingbox(data_dict)
+        box_dict = box_group_and_SE3(bbox_dict, self._root_path, log_id, 1)
+        track_group = box_group_to_track_group(box_dict)
+        track_group_tensors_dict, _ = label_to_tensor(track_group, 1)
+        gt_boxes = list(track_group_tensors_dict.values())[0]
+        gt_boxes = gt_boxes.squeeze(1).numpy()
+
+        info = {}
+        res = {
+            "lidar": {
+                "type": "lidar",
+                "points": points,
+                "annotations": {
```

Comment:
is there any reason they assume annotations are None here? https://github.com/hchittanuru3/CenterPoint/blob/328317fc3dbb0d55f1f89264e10fed2a59fdf226/det3d/datasets/nuscenes/nuscenes.py#L167